R/add_keep_population_flag.R

```diff
@@ -0,0 +1,146 @@
+#' Add keep_popluation flag
+#'
+#' @description Add keep_population flag to individual files
+#' @param data A data frame
+#'
+#' @return A data frame with keep_population flags
+#' @family individual_file
+
+add_keep_population_flag <- function(individual_file, year, chi_var_name = "chi") {
```

Comment:
```suggestion
add_keep_population_flag <- function(individual_file, year") {
```
I dont think we need to specify the chi variable here as this should still be `chi` at this stage in the processing. However i know why you may have added this here if you were testing an existing individual file.

Reply:
Sure